docs/docsite/rst/user_guide/windows_winrm.rst

```diff
@@ -341,8 +341,8 @@ Some system dependencies that must be installed prior to using Kerberos. The scr
 
 .. code-block:: shell
 
-    # Via Yum (RHEL/Centos/Fedora)
-    yum -y install gcc python-devel krb5-devel krb5-libs krb5-workstation
+    # Via dnf (RHEL/Centos/Fedora)
```

Comment:
I would rather keep both YUM and DNF commands and annotate them using different versions of RHEL/CentOS/Fedora.

Reply:
A couple of comments/questions:
1. These docs are for ansible-core 2.13 and later - Are there any older (pre dnf) versions of RHEL/CentOS/Fedora that will actually work with core 2.13 (especially given the python 3.8 requirement)?
2. If so, will y'all maintain this list going forward? The risk of adding older version details to include yum is that it can get outdated quite fast.

Reply:
> I would rather keep both YUM and DNF commands and annotate them using different versions of RHEL/CentOS/Fedora.

Could you make an example please? What version of RHEL/Centos/Fedore should we specify for the old yum python-devel? And what for the new dnf python3-devel?

Reply:
I would see something - 

```console
# via Yum (RHEL/Centos/Fedora for the older version)
yum -y install gcc python-devel krb5-devel krb5-libs krb5-workstation

or 

# via DNF (RHEL/Centos/Fedora for the newer version)
dnf -y install gcc python3-devel krb5-devel krb5-libs krb5-workstation

```

Reply:
Commit updated!